.github/workflows/wheel.yml

```
@@ -5,11 +5,19 @@ on:
     branches: [ master ]
     tags:
       - 'v*'
-  pull_request:
+  pull_request_target:
     branches: [ master ]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ matrix.os }}
```

actionlint

context "matrix" is not allowed here. available contexts are "github", "inputs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)